fbw-a32nx/src/systems/instruments/src/OANC/Components/ContextMenu.scss

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2021-2023 FlyByWire Simulations
```

Comment:
Should these all really be claiming 2021-2023?

(this applies to multiple new files in the PR)

Reply:
fixed all of them now